src/SpectatorView.Native/README.md

```diff
@@ -33,6 +33,26 @@ If you are using an Elgato capture card, you will need to clone Elgato's [gameca
 - Update the Elgato_Filter user macro in `dependencies.props` with the corresponding path on your computer.
 - Restart Visual Studio
 
+**Azure Kinect**
+If you are using an Azure Kinect, you will need to download or clone the Azure Kinect Sensor SDK with [Microsoft installer](https://download.microsoft.com/download/e/6/6/e66482b2-b6c1-4e34-bfee-95294163fc40/Azure%20Kinect%20SDK%201.3.0.exe) or [github repo](https://github.com/microsoft/Azure-Kinect-Sensor-SDK)
+
+If cloning from Github: 
+- Open a Command Prompt in administrator mode
+- Navigate to a folder in which you would like to store your repositories (ex: c:\git)
+- git clone <https://github.com/microsoft/Azure-Kinect-Sensor-SDK>
+- Update the AzureKinectSDK user macro in `dependencies.props` with the corresponding path on your computer.
+- Restart Visual Studio
+
+**Azure Kinect Body Tracking**
+If you are using Azure Kinect and want to utilize body tracking dependent features (ie. people-only occlusion), you will you will need to download or install the Azure Kinect Body Tracking SDK here: [Windows download links](https://docs.microsoft.com/en-us/azure/kinect-dk/body-sdk-download)
+
+If installing from Nuget: 
+- Open a Command Prompt in administrator mode
+- Navigate to a folder in which you would like to store your repositories (ex: c:\git)
+- Install-Package Microsoft.Azure.Kinect.BodyTracking -Version 1.0.0 
```

Comment:
>Install-Package [](start = 2, length = 15)

Install-Package is a PowerShell command, right? Does the first step need to say "Open a PowerShell prompt" instead of Command Prompt?